guides/assets/stylesheets/main.rtl.css

```diff
@@ -0,0 +1,762 @@
+/* Guides.rubyonrails.org */
+/* Main.css */
```

Comment:
Although it would be copied to `main.css`, it may be useful to maintain here that it is in fact `Main.rtl.css`.